lib/config/fileconf.go

```diff
@@ -92,6 +92,9 @@ type FileConfig struct {
 
 	// Okta is the "okta_service" section in the Teleport configuration file
 	Okta Okta `yaml:"okta_service,omitempty"`
+
+	// PluginService is the section of the config for configuring the plugin service.
+	PluginService PluginService `yaml:"plugin_service,omitempty"`
```

Comment:
Nit: None of the other services have "service" suffix so I suggest we rename this to just "Plugins" as well for naming consistency.
```suggestion
	// Plugins is the section of the config for configuring the plugin service.
	Plugins PluginService `yaml:"plugin_service,omitempty"`
```